docs/manual/creating-a-checker.tex

```diff
@@ -477,8 +477,8 @@
 % \sunjavadocanno{java.base/java/lang/annotation/Target.html}{Target}.)
 
 The annotations should be placed within a directory called \<qual>, and
```

Comment:
Maybe split this up into two: the qualifiers should be in a `qual` sub-package of the type checker subpackage - that's were the checker reflectively looks and the only thing that matters for that is the package name.
As mentioned in `{creating-parts-of-a-checker}` the files for that `qual` package can be either in a subdirectory or in a separate gradle module, depending on whether they should be distributable separately. Briefly repeat that point.

Reply:
Thanks, updated.